src/data_retrieval/__main__.py

```diff
@@ -30,13 +32,11 @@
 
 def get_cds_ecmwf():
     logger.info("Downloading Copernicus CDS data of ECMWF global forecast..")
-    file_name: str = "ecmwf_forecast_global_all_years.grib"
-    output_path: str = os.path.expanduser("~/Downloads/ecmwf_global_forecast")
-    setup_output_path(output_path)
+    blob_path = "/raw/glb/ecmwf/ecmwf-monthly-seasonalforecast-1981-2023.grib"
 
     # Utilizing the specific range of available years you've provided
     # Updated to include 1981 through 2023
-    available_years: List[int] = list(range(1981, 2024))
+    available_years: List[int] = list(range(1981, 2023))
```

Comment:
This is wrong `range(1981, 2023)` will return values up to 2022. The last value in the range is not included.
Please change it back to `range(1981, 2024)` so we also have the year 2023